pkg/ext/fetcher_ext_client.go

```diff
@@ -34,6 +34,12 @@ type AdminFetcherExtInterface interface {
 
 	// FetchTaskVersion fetches particular version of task in a  project, domain
 	FetchTaskVersion(ctx context.Context, name, version, project, domain string) (*admin.Task, error)
+
```

Comment:
we can rename all these files to be simply
fetcher.go
deleter.go
etc
as they are already in the go package `ext`?

Reply:
Done